lib/migrator.js

```diff
@@ -58,6 +58,35 @@ module.exports = class Migrator {
     return templateFilePaths;
   }
 
+  skipTemplatesUsedAsPartial(templateFilePaths) {
+    const componentsWithPartial = getPartialTemplates(this.findTemplates());
+
+    if (componentsWithPartial.length) {
+      componentsWithPartial.sort().forEach(component => {
+        console.info(`❌ Did not move '${component}' due to usage as a "partial"`);
+      });
+
+      templateFilePaths = templateFilePaths.filter(templateFilePath => {
+        // Extract '/app/templates/components/nested1/nested-component.hbs'
+        let filePathFromApp = templateFilePath.slice(this.projectRoot.length);
+
+        if (/\/\-[\w\-]+\.hbs/.test(filePathFromApp)) {
```

Comment:
I'm not sure what lines 73-75 were for. This check didn't appear in `skipTemplatesUsedAsLayoutName` and wasn't evaluated to be `true` in tests.

Reply:
When Ember sees `{{partial "foo"}}` it will look in to locations:

* `app/templates/foo.hbs`
* `app/templates/-foo.hbs`

I think this is attempting to identify when `-foo.hbs` format is used, and still match `{{partial "foo"}}`?